packages/blade/src/components/ChatBubble/index.ts

```diff
@@ -0,0 +1 @@
+export * from './ChatBubble';
```

Comment:
Only export the expected components and types instead of exporting all from here